src/test/java/com/basho/riak/client/core/DefaultNodeManagerTest.java

```diff
@@ -158,12 +158,13 @@ public void addNode()
         assertEquals(mockNodes.size() + 1, healthy.size());
     }
 
-    private class IsException extends ArgumentMatcher<Exception>
```

Comment:
@nickkeers - so the type signature says - I'm creating an argument matcher which will match sub-classes of exception.